serialization/physics.proto

```diff
@@ -89,6 +89,12 @@ message DiscreteTrajectory {
   repeated int32 fork_position = 3;
   // Added in 陈景润.
   optional Downsampling downsampling = 4;
+
+  // ZFP support added in Frobenius.
+  optional int32 zfp_codec_version = 7;
+  optional int32 zfp_library_version = 8;
+  optional bytes zfp_timeline = 5;
+  optional int32 zfp_timeline_size = 6;
```

Comment:
These should be wrapped in a message, since they are really optional and none or all should be present.

Reply:
Done, PTAL.